### PR TITLE
Update parity 2.5.12, default testnet is Görli now.

### PR DESCRIPTION
**Pull request by kurotaky**
Added support for using parity 2.5.12, default testnet is Görli now.

### CODE REVIEW

File: PREREQUISITES.md [hunk 1]
```diff
@@ -60,13 +60,16 @@ There is a rake task to run node on testnet network, that you can run from your
...
     $ rake ethereum:node:test
 
-It will run parity node, unlock the first account on the account list, but you need to supply it with password. 
+It will run parity node, unlock the first account on the account list, but you need to supply it with password.
 To do that adding create file containing password accessable from your parity folder, which should be one of the following:
  * `/Users/You/AppData/Roaming/Parity/Ethereum` on Windows
  * `/Users/you/Library/Application Support/io.parity.ethereum` on MacOS
  * `/home/you/.local/share/parity` on Linux/Unix
- * `/home/you/.parity` on Linux and MacOS for Parity versions older then 1.5.0
+ * `/home/you/.parity` on Linux and MacOS for Parity versions older then 2.5.0
```
Comment (marekkirejczyk) [inline PREREQUISITES.md]:
```suggestion
 * `/home/you/.parity` on Linux and MacOS for Parity versions older then 1.5.0
```

**Comment by marekkirejczyk** [comment]
Awesome work! ❤️❤️❤️

**Comment by marekkirejczyk** [comment]
Published in 2.3

**Comment by kurotaky** [comment]
Thanks! :)

**Pull request merged**